lib/handlers/products.js

```diff
@@ -1,77 +1,7 @@
 'use strict';
```

Comment:
We can remove products from the cloudapi, this was mainly added before we created the central navigation